package.json

```diff
@@ -64,6 +65,7 @@
     "memoizee": "^0.4.17",
     "pinejs-client-core": "^8.1.4",
     "randomstring": "^1.3.1",
+    "string-hash": "^1.1.3",
```

Comment:
@Page- are we 100% sure we want to go with this module? Altough it is extensively used, the repo has been deprecated/made read-only:
https://github.com/darkskyapp/string-hash/blob/master/index.js

Do you consider using native crypto.createHash and create a simple md5 or sha hash too problematic/slow here?

Reply:
Also library mantainer said library is no longer mantained here: https://github.com/darkskyapp/string-hash/issues/12#issuecomment-619037758 And deleted gh account. Altough it is a simple module in this same issue they are discussing possible optimisations etc which I would hope are baked in the native module

Reply:
It's the module we use in odata-to-abstract-sql which is why I reused it, I've checked for alternatives and gone with fnv1a by sindresorhus as it seems to be a good mix of speed/support - we don't need a cryptographically secure hash and speed is more important